templates/pkg.tmpl

```diff
@@ -48,8 +48,12 @@
     <div class="tile tile-centered divided">
       <div class="tile-content">
         <div class="tile-title">
-          <figure class="avatar avatar-xs" data-initial="" style="background-color: #5764c6;">
-          </figure>
+          <picture>
+            <!-- if Mobile 25px PNG else 99px PNG -->
+            <source srcset="https://github.com/${pkg["github_owner"].str}.png?size=99" type="image/png" media="(min-width: 767px)">
+            <img class="avatar avatar-xl" src="https://github.com/${pkg["github_owner"].str}.png?size=25" alt="${pkg["github_owner"].str}'s avatar">
```

Comment:
IMO it's better to just keep the avatar small on all devices.

Reply:
oh and btw <picture> is not supported AT ALL on IE.

Reply:
:+1:

Reply:
AFAIK `<picture>` should fallback to the existent `<img>` anyways...